transit_odp/browse/lta_column_headers.py

```diff
@@ -112,3 +112,121 @@ def get_operator_name(otc_service: dict) -> str:
+        if otc_service.get("dataset_id")
+        else "Unpublished",
+    ),
+    (
+        "Registration Status",
+        lambda otc_service: "Registered"
+        if otc_service.get("otc_licence_number")
+        else "Unregistered",
+    ),
+    (
+        "Scope Status",
+        lambda otc_service: OTC_SCOPE_STATUS_OUT_OF_SCOPE
+        if otc_service.get("scope_status", False)
+        else OTC_SCOPE_STATUS_IN_SCOPE,
+    ),
+    (
+        "Seasonal Status",
+        lambda otc_service: get_seasonal_service_status(otc_service)
+        if otc_service.get("seasonal_status") is not None
+        else "Not Seasonal",
+    ),
+    ("Timeliness Status", lambda otc_service: otc_service.get("staleness_status")),
+    (
+        "Organisation Name",
+        lambda otc_service: get_operator_name(otc_service)
+        if otc_service.get("operator_name") is None
+        or otc_service.get("operator_name") == ""
+        else otc_service.get("operator_name"),
+    ),
+    ("Data set ID", lambda otc_service: otc_service.get("dataset_id")),
+    ("XML:Filename", lambda otc_service: otc_service.get("xml_filename")),
+    (
+        "XML:Last Modified Date",
+        lambda otc_service: otc_service.get("last_modified_date"),
+    ),
+    (
+        "XML:Operating Period End Date",
+        lambda otc_service: otc_service.get("operating_period_end_date"),
+    ),
+    (
+        "Date Registration variation needs to be published",
+        lambda otc_service: otc_service.get("effective_stale_date_otc_effective_date"),
+    ),
+    (
+        "Date for complete 42-day look ahead",
+        lambda otc_service: get_42_day_look_ahead_date(),
+    ),
+    (
+        "Date when data is over 1 year old",
+        lambda otc_service: otc_service.get("effective_stale_date_last_modified_date"),
+    ),
+    (
+        "Date seasonal service should be published",
+        lambda otc_service: otc_service.get("effective_seasonal_start_date"),
+    ),
+    ("Seasonal Start Date", lambda otc_service: otc_service.get("seasonal_start")),
+    ("Seasonal End Date", lambda otc_service: otc_service.get("seasonal_end")),
+    (
+        "Registration:Operator Name",
+        lambda otc_service: otc_service.get("otc_operator"),
+    ),
+    (
+        "Registration:Licence Number",
+        lambda otc_service: otc_service.get("otc_licence_number"),
+    ),
+    (
+        "Registration:Service Type Description",
+        lambda otc_service: otc_service.get("otc_service_type_description"),
+    ),
+    (
+        "Registration:Variation Number",
+        lambda otc_service: otc_service.get("otc_variation_number"),
+    ),
+    (
+        "Registration:Start Point",
+        lambda otc_service: otc_service.get("otc_start_point"),
+    ),
+    (
+        "Registration:Finish Point",
+        lambda otc_service: otc_service.get("otc_finish_point"),
+    ),
+    ("Registration:Via", lambda otc_service: otc_service.get("otc_via")),
+    (
+        "Registration:Granted Date",
```

Comment:
**Registration:Start Point**, **Registration:Finish Point**, **Registration:Via** and **Registration:Granted Date** are no longer needed in this report (comment on ticket)

Reply:
done